src/redis-cli.c

```diff
@@ -2280,6 +2288,8 @@ clusterManagerCommandDef clusterManagerCommands[] = {
      "from <arg>,copy,replace"},
     {"backup", clusterManagerCommandBackup, 2,  "host:port backup_directory",
      NULL},
+    {"masters-only", NULL, 0, NULL, "only-masters"},
+    {"slaves-only", NULL, 0, NULL, "only-slaves"},
```

Comment:
The first member of `clusterManagerCommandDef` is the command name and "masters-only" is not a command. In this case, the command is `call`. Please take a look at how help is implemented for other commands.
You should specify `only-masters` and `only-masters` as options for the `call` command.

If I launch `redis-cli --cluster help` with your PR, I'll see this output:

```
  call           host:port command arg arg .. arg
  set-timeout    host:port milliseconds

  ...

  masters-only
                 --cluster-only-masters
  slaves-only
                 --cluster-only-slaves
  help
```

But it should be

```
  call           host:port command arg arg .. arg
                     --cluster-only-masters
                     --cluster-only-slaves
 
  ...

  help
```

Reply:
fixed..